modd.conf

```diff
@@ -0,0 +1,4 @@
+command/*.go !**/*_test.go  {
+    prep: make dev
+    daemon +sigterm: bin/nomad agent -dev -bind 0.0.0.0 -log-level INFO
+}
```

Comment:
Do we need this file committed, or was this for testing?

Reply:
Ups, my bad! I use it for local development. Removing it!